### PR TITLE
Add Flowis protocol decoder

### PR DESCRIPTION
**Pull request by merbanan**

### CODE REVIEW

File: src/devices/flowis.c [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+static char *output_fields[] = {
+        "model",
+        "id",
+        "type",
+        "amount",
+        "mic",
+        NULL,
+};
+
+r_device flowis = {
```
Comment (zuckschwerdt) [inline src/devices/flowis.c:113]: r_device **const** flowis
;)

Comment (merbanan) [reply src/devices/flowis.c]: fixed

**Comment by merbanan** [comment]
Any similarities with marlec_solar.c is purely coincidental.

**Comment by merbanan** [comment]
@zuckschwerdt should be ready to commit now.

**Comment by zuckschwerdt** [comment]
Looks good. Should "volume" maybe have a suffix "volume_m3", and should "timestamp" indicate it's device related and not metadata, e.g. something like "device_time" or "sensor_clock"?

**Comment by zuckschwerdt** [comment]
Sorry, I just merged two decoders, so I pushed a fix here.

**Comment by merbanan** [comment]
I'll rebase and fix when we are happy with the keys.

**Comment by merbanan** [comment]
We have volume_gallon or something like that. Yes it should be _m3.

**Comment by merbanan** [comment]
And device_time sounds fine.

**Pull request merged**